actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
> For that kind of hackery, you could point to the uncompiled action_cable.coffee for inserting your console.logs, etc.

On npm most would prefer the JS/ES6 version. Even if they would use coffee version we don't ship with it right now on npm.

> That's true if your app compresses assets, but it's not a given. Especially considering people can install via npm and use action_cable.js outside of Rails / Sprockets.

Anyone doing doing this will already be using compression by some other means.

Here's how our current npm package looks- 
- https://monosnap.com/file/RdcH7bbAYS0dWP7N2FoX2JCkMflK3J

Here's some other examples from popular libraries- 
- https://monosnap.com/file/QaCy3QX8z0ZIGqBUAAVTEc8jc6EbTQ
- http://take.ms/zLdpW

Most of them contain uncompressed + compressed version. Its ok to just ship with compressed one, since its essential for things listed by Matthew:
- sourcemaps
- edit/check source code of actual library
- source linking of methods, etc in editors

> I don't feel strongly that it should be compressed given its relatively small file size, but it feels like the right thing to do for a distributed .js file

As seen above many libraries ship with both, so that they can be used as needed. For us its ok to ship just with unminified version as well.